src/action_creators/EnrollActionCreator.js

```diff
@@ -11,7 +12,50 @@ const EnrollActionCreator = {
     return {
       type: ActionTypes.TOGGLE_PREFERENCES
     }
+  },
+
+  requestStudentEnrollment(student) {
+    var that = this;
```

Comment:
But I don't like this piece of code too. 
I just don't want the arrow functions all inside the `then(...)` callback